doc/level0.rst

```diff
@@ -92,26 +92,150 @@ To solve this problem in a better way, bi-directional communication
-  - attachment? something else?
-- message text
+For proper support of a multi-device scenario, it is necessary to have
+bi-directional communication between different MUAs. This is possible
+e.g. via access to a shared IMAP mailbox. Because of the complexity of
+this approach however, multi-device support in the sense of devices
+coordinating with each other is out of scope for Autocrypt Level 0. It
+is still important to avoid "lock-in" of secret key material on a
+particular client. For this reason, Autocrypt includes a way to
+"export" keys (and possibly other state) for other clients to pick up,
+asynchronously and with explicitly required user interaction.
+
+The mechanism available in Autocrypt level 0 is a specially-formatted
+e-mail message called the Autocrypt Setup Message.  An
+already-configured Autocrypt client can generate an Autocrypt Setup
+Message, and send it to itself.  A not-yet-configured Autocrypt client
+(a new client in a multi-device case, or recovering from device
+failure or loss) can import the Autocrypt Setup Message and recover
+the ability to read existing messages.
+
+An Autocrypt Setup Message is protected with a strong Setup Code.
+
+Message Structure
+++++++
+
+The Autocrypt Setup Message itself is an e-mail message with a
+specific format, which contains a payload protected by the setup code.
+
+- Both the To and From headers MUST be the address of the user.
+- The Autocrypt Setup Message MUST have a `multipart/mixed` structure,
+  with the first part a `text/plain` or `text/html` part that provides
+  a human-readable description to the user about the purpose of the
+  message.
+- The "payload" of the Autocrypt setup message is a MIME part that is
+  a direct child of the root part, with a Content-Type of
+  `application/pgp-key-backup`.  There MUST be exactly one payload
+  part of this Content-Type in the message.
+- The payload MUST contain a single ASCII-armored block of OpenPGP
+  symmetrically encrypted data, and MAY include other text above or
+  below the ASCII-armored data, which MUST be ignored while
+  processing.
+- Decrypting the payload should produce an ASCII-armored OpenPGP
+  transferable secret key.
+- The encryption algorithm used MUST be at least as strong as 128-bit
+  symmetric encryption.  It SHOULD be AES-128.  The passphrase MUST be
+  the Setup Code (see below), used under OpenPGP's salted+iterated S2K
+  algorithm.
+
+The payload MIME part MAY optionally use the OpenPGP Transfer Format
+(see :ref:`openpgp-transfer-format`) for improved usability.
+
+Setup Code
++++++
+
+The setup code MUST be generated by the implementation itself using a
+CSPRNG, and presented directly to the user for safekeeping. It MUST
+NOT be included in the Autocrypt Setup Message, or otherwise
+transmitted over e-mail. Its format SHOULD be 24 uppercase
+alphanumeric characters, divided into six blocks of four, separated by
+dashes. The dashes are part of the secret code. This format holds 124
```

Comment:
some of us think it's better to use simply numbers which would amount to roughly 9 * 4 number blocks.   Some user studies (ask Dominik/vincent) indicate that using numbers produces the least confusion/errors when they are asked to type it in.  Also avoiding latin characters probably helps with chinese and other non-latin based alphabets. 

Maybe we can also use an error correction number block so that we have 10 * 4 numbers and a user can mistype 2 or 3 numbers.